cmd/helm/template_test.go

```diff
@@ -138,9 +138,9 @@ func TestTemplateCmd(t *testing.T) {
 		{
 			name:        "check_name_template",
 			desc:        "verify --name-template result exists",
-			args:        []string{subchart1ChartPath, "--name-template", "foobar-{{ b64enc \"abc\" }}-baz"},
```

Comment:
same here; can we add a test case that demonstrates `foobar-{{ b64enc \"abc\" }}-baz` should fail rather than removing it?

Reply:
thanks for your advice. already added them.